.github/workflows/pytest.yml

```diff
@@ -0,0 +1,86 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu:
+
+    runs-on: ubuntu-18.04
```

Comment:
Now that this doesn't require the pyrfr (and the pyrfr being available as a wheel), can we relax this?